Start of 1.11.2 port

TheIcyOne opened the pull request:
ALPHA ALPHA ALPHA


Seriously don't use this if you value your world.

TheIcyOne committed:
Merge remote-tracking branch 'refs/remotes/Growlith1223/1.10.2' into 1.10.2

TheIcyOne committed:
Merge remote-tracking branch 'refs/remotes/Growlith1223/1.10.2' into 1.10.2

TheIcyOne committed:
Preliminary work on 1.11.2 port

Just because you can load into a world, doesn't make this a playable
version. By all means try, I like to see the crash reports.

TheIcyOne committed:
Merge remote-tracking branch 'refs/remotes/Growlith1223/1.10.2' into 1.11.2

TheIcyOne committed:
1.11 pls

TheIcyOne committed:
Items pls

TheIcyOne committed:
Fix issues with nonliving views

Legitimately surprised there hasn't been a similar issue before.

TheIcyOne committed:
Turns out that should be in the libs directory

TheIcyOne committed:
Lowercased a load of filenames.

No more missing textures.

Hopefully.

Valchap commented:
I tried to build it but i can't, an error occurred. Is this normal because alpha ?

Growlith1223 commented:
yes

On Apr 24, 2017 12:46 PM, "Valentin" <notifications@github.com> wrote:

> I tried to build it but i can't, an error occurred. Is this normal because
> alpha ?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Growlith1223/ArsMagica2/pull/301#issuecomment-296766743>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFC9gbr8D2u4GNmilHG57JqMDwbzKIUvks5rzOAHgaJpZM4MvBkf>
> .
>

TheIcyOne committed:
Fixed cloaks for 1.11

For some reason, this worked in the past. I really don't know how or
why.

TheIcyOne commented:
As a general rule, assume that any issues you run into when trying to build it are my fault for now.

TheIcyOne committed:
More assorted fixes

Finally worked out why containers were borked.

Of course it was a load of null ItemStacks.

TheIcyOne committed:
3 down, about 20 to go

Can you tell how much I love 1.11.2's ItemStack changes?

TheIcyOne committed:
Should now actually be playable

I'm going to go have a nap.

TheIcyOne commented:
From this point forward, assume that any issues you run into when trying to build it may not actually be my fault.

TheIcyOne committed:
Calefactor now extends AMPoweredContainer

Muh filesizes

TheIcyOne committed:
Added scary scary warning text

It's probably not going to destroy any worlds that 1.10.2 didn't, but at
least I have an excuse if anybody builds it for now.

Valchap commented:
Does the compilation work for you?
I have an error with "ItemStack.setItem" method in bound items, this method no longer exists in 1.11.

TheIcyOne commented:
Ah, that's still my fault. I forgot they existed.

TheIcyOne commented:
I blame the Bound Shield for being basically irrelevant.

TheIcyOne committed:
I didn't forget any bound items

None at all. These are just.... cosmetic changes.

xenxero1 commented:
I love the idea of the bound shield and all, but the thing right now is buggy as hell and basically non-functional.

TheIcyOne committed:
Updating mappings

Doing it now so it doesn't mess me up when I actually go to build.

TheIcyOne committed:
Handling of biomes with no types

I still blame the other mod.

TheIcyOne committed:
Dealing with summon caps

Bandaid on a bandaid. Also includes burnout fixes because it happens to
be in the EntityExtension at the same time.

TheIcyOne committed:
We can now deal with the weird lefties

Yay. (Needs backporting to 1.10.2 versions at some point)

TheIcyOne committed:
Fixed summons

Technically this was a one-line fix, but I've also added clearing of
summons on login/out, so that missing summons don't fill the cap
forever.

TheIcyOne committed:
oops

TheIcyOne committed:
Changes for build

Should be possible to build the repo now.

SjetCraft committed:
Merge remote-tracking branch 'refs/remotes/origin/1.11.2' into 1.10.2

SjetCraft committed:
Merge branch 'refs/heads/1.10.2' into 1.11.2

SjetCraft committed:
Main Texture Refactoring

Refactored textures and assests.
Added 2 missing textures.
Fixed Essence Conduit, Crystal Marker and Calefactor placing

SjetCraft committed:
Texture bug squash

Squashed last texture bugs.
Textures are now at 1.10 parity

TheIcyOne committed:
Merge pull request #1 from SjetCraft/1.11.2

1.11.2 Texture Update

TheIcyOne commented:
At this point, it's buildable, and provides a playable experience approximately as buggy as 1.10 versions.

TheIcyOne committed:
light pls

replaced deprecated functions

TheIcyOne committed:
ok seriously now git

Given this commit, I can build a playable version from a clean repo.
